libass/ass_fontselect.c

```diff
@@ -1135,3 +1135,12 @@ void ass_map_font(const ASS_FontMapping *map, int len, const char *name,
         }
     }
 }
+
+void ass_update_embedded_fonts(ASS_Library *lib, ASS_FontSelector *selector, FT_Library ftlib)
+{
+    if (!selector->embedded_provider)
+        return;
+    for (size_t i = lib->num_fontsloaded; i < lib->num_fontdata; ++i)
```

Comment:
If you’re switching the increments to postfix, you’ve missed this one.

Reply:
Done, thx.